.github/workflows/main.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - run: cargo xtask install
       - run: cargo xtask gen-receipt
-      - run: cargo test -F $FEATURE -F profiler
+      - run: cargo test -F $FEATURE -F profiler -F test-exact-cycles
```

Comment:
We won't be able to run tests with `-F test-exact-cycles` on all runners, just on the ones with the `docker` tag.